lib/oli_web/live/sections/overview_view.ex

```diff
@@ -160,7 +160,7 @@ defmodule OliWeb.Sections.OverviewView do
             ><span>Preview Course as Instructor</span> <i class="fas fa-external-link-alt self-center ml-1" /></a>
           </li>
           <li><a
-              href={Routes.page_delivery_path(OliWeb.Endpoint, :index_preview, @section.slug)}
+              href={Routes.page_delivery_path(OliWeb.Endpoint, :index, @section.slug)}
```

Comment:
This change reverts the fix made on [MER-1973](https://eliterate.atlassian.net/browse/MER-1973)

Reply:
Yes, we need this link to be the link that routes to actual student view, not a preview.